source/CMakeLists.txt

```diff
@@ -57,6 +57,21 @@ FOREACH(build ${DEAL_II_BUILD_TYPES})
   #
 
   GET_PROPERTY(_objects GLOBAL PROPERTY DEAL_II_OBJECTS_${build})
+
+  #
+  # FIXME: Somehow the cuda object files lose the "generated" property.
+  # This is an ugly hack
+  #
+  FOREACH(_object ${_objects})
+    IF("${_object}" MATCHES "cu.o$")
+      SET_SOURCE_FILES_PROPERTIES("${_object}"
+        PROPERTIES
+        EXTERNAL_OBJECT TRUE
+        GENERATED TRUE
+        )
+    ENDIF()
+  ENDFOREACH()
```

Comment:
@kalj Is this a known problem with cmake and Cuda or how did you find out about this?